### PR TITLE
React to ANCM requestTimeout change for 2.1

### LINKED ISSUE #5264: Change ANCM requestTimeout property to remove 'requestTimeout must be specified in whole minutes'
The requestTimeout in the aspnetcore_schema will be updated to allow a requestTimeout to be specified in seconds. https://github.com/aspnet/Docs/blob/313486d76f89ceac7af9b8c0769f72d1b224d4ca/aspnetcore/host-and-deploy/aspnet-core-module.md can now remove the note about only allowing whole minutes.
cc/ @shirhatti

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #5264 

[Internal Review Topic](https://review.docs.microsoft.com/en-us/aspnet/core/host-and-deploy/aspnet-core-module?view=aspnetcore-1.0&branch=pr-en-us-5949)

In the <=2.0 and >=2.1 rendered table content, you only need to look at the `requestTimeout` row *Description* column content.

### CODE REVIEW

File: aspnetcore/host-and-deploy/aspnet-core-module.md [hunk 1]
```diff
@@ -60,18 +60,36 @@ See [Sub-application configuration](xref:host-and-deploy/iis/index#sub-applicati
...
 | `stdoutLogEnabled` | <p>Optional Boolean attribute.</p><p>If true, **stdout** and **stderr** for the process specified in **processPath** are redirected to the file specified in **stdoutLogFile**.</p> | `false` |
 | `stdoutLogFile` | <p>Optional string attribute.</p><p>Specifies the relative or absolute file path for which **stdout** and **stderr** from the process specified in **processPath** are logged. Relative paths are relative to the root of the site. Any path starting with `.` are relative to the site root and all other paths are treated as absolute paths. Any folders provided in the path must exist in order for the module to create the log file. Using underscore delimiters, a timestamp, process ID, and file extension (*.log*) are added to the last segment of the **stdoutLogFile** path. If `.\logs\stdout` is supplied as a value, an example stdout log is saved as *stdout_20180205194132_1934.log* in the *logs* folder when saved on 2/5/2018 at 19:41:32 with a process ID of 1934.</p> | `aspnetcore-stdout` |
+::: moniker-end
+::: moniker range=">= aspnetcore-2.1"
+[!INCLUDE[](~/includes/2.1.md)]
```
Comment (jkotalik) [inline aspnetcore/host-and-deploy/aspnet-core-module.md]: Any way to minimize the repetition here?

Comment (guardrex) [reply aspnetcore/host-and-deploy/aspnet-core-module.md]: How do you mean ... the `moniker ranges` ... no can do unfortunately. They don't work inside table cells AFAIK.

Comment (jkotalik) [reply aspnetcore/host-and-deploy/aspnet-core-module.md]: Yeah that's what I meant.

Comment (Rick-Anderson) [reply aspnetcore/host-and-deploy/aspnet-core-module.md]:
```Markdown
::: moniker range="moniker-range-expression"

Versioned Content

::: moniker-end
```

A versioned zones starts with `::: moniker` on a new line and specifies a `range` property, the value of which is a valid moniker range. The zone ends with `::: moniker-end`, on its own line. All content between these two markers applies to the versions matched by the `range` expression.

### Limitations of Zones

1. You **can** use includes inside of a zone, along with any other standard Markdown or Docs-specific Markdown extensions.
1. You **can** use a zone inside of other constructs, like a blockquote or note.
1. You **cannot** use a `moniker` zone inside of another `moniker` zone. Nesting is not permitted.

Comment (guardrex) [reply aspnetcore/host-and-deploy/aspnet-core-module.md]: ... so they don't really say explicitly if we can inject them into a table. Has it been tested ... I can test it on this PR if you want.

```
| Col1 | Col2 |
| ---- | ---- |
| Val1 | Val2 |
::: moniker range="<= aspnetcore-1.1"
| Val1 | Val2 |
::: moniker-end
::: moniker range=">= aspnetcore-2.0"
| Val1 | Val2 |
::: moniker-end
| Val1 | Val2 |
```

Comment (Rick-Anderson) [reply aspnetcore/host-and-deploy/aspnet-core-module.md]: @guardrex  try that. Currently the entire table is missing when I select 2.0

**Comment by guardrex** [comment]
@Rick-Anderson Try it now (that didn't throw a build error) ... [Internal Review Topic](https://review.docs.microsoft.com/en-us/aspnet/core/host-and-deploy/aspnet-core-module?view=aspnetcore-1.0&branch=pr-en-us-5949)

WRT it not showing for 2.0 before ... rookie first-timer mistake. I know why it was broken; so if we end up going back to my original two-table approach, it should work.

**Comment by Rick-Anderson** [comment]
@guardrex  breaks table rendering. I think I'll squash/merge and open a bug.

image

**Comment by Rick-Anderson** [comment]
@guardrex  can you move he `requestTimeout` row to the bottom of the table? You could have a he `requestTimeout` row that said, see the `requestTimeout` row at the bottom of the table.

**Comment by Rick-Anderson** [comment]
@guardrex  I think it's OK that it breaks the table rendering for now - especially if you move it to the bottom of the table.

**Comment by guardrex** [comment]
I'd prefer not to do it based on aesthetics. Why not just two tables (I fixed the earlier 2.0 problem)? [Internal Review Topic](https://review.docs.microsoft.com/en-us/aspnet/core/host-and-deploy/aspnet-core-module?view=aspnetcore-1.0&branch=pr-en-us-5949)

**Comment by Rick-Anderson** [comment]
I'm thinking they can fix the bug fairly quickly. If not, use 2 tables.

**Comment by guardrex** [comment]
... also my table is *alphabetical*. It would upset the world order ... **_Earth may fly off it's axis into the Sun!_**

I wouldn't want the deaths of billions of humans on my hands. :grinning:

**Comment by guardrex** [comment]
I reverted it ... see if it works now.

File: aspnetcore/host-and-deploy/aspnet-core-module.md [hunk 2]
```diff
@@ -60,18 +60,36 @@ See [Sub-application configuration](xref:host-and-deploy/iis/index#sub-applicati
...
```
Comment (Rick-Anderson) [inline aspnetcore/host-and-deploy/aspnet-core-module.md]: Can you remove [!INCLUDE[](~/includes/2.1.md)] ? It's confusing at the top of the table.

**Pull request merged**